MPF.Core/InfoTool.cs

```diff
@@ -1382,12 +1382,22 @@ public static bool WriteProtectionData(string? outputDirectory, string? filename
                     path = Path.Combine(outputDirectory, $"!protectionInfo{filenameSuffix}.txt");
 
                 using var sw = new StreamWriter(File.Open(path, FileMode.Create, FileAccess.Write));
-                foreach (var kvp in info.CopyProtection.FullProtections)
+
+                List<string> sortedKeys = [.. info.CopyProtection.FullProtections.Keys.OrderBy(k => k)];
+                foreach (string key in sortedKeys)
                 {
-                    if (kvp.Value == null)
-                        sw.WriteLine($"{kvp.Key}: None");
+                    string scanPath;
+                    if (hideDriveLetters)
+                        scanPath = key.Substring((Path.GetPathRoot(key) ?? String.Empty).Length);
+                    else
+                        scanPath = key;
+
+                    List<string>? scanResult = info.CopyProtection.FullProtections[key];
+
+                    if (scanResult == null)
+                        sw.WriteLine($"{Path.DirectorySeparatorChar}{scanPath}: None");
```

Comment:
Since you're prepending the directory separator character here where it previously wasn't, why not just include that in the `if (hideDriveLetters)` block instead?

Reply:
Oops, good catch, that shouldn't be added if drive letters are not hidden